src/Service/PostcodeService.php

```diff
@@ -187,6 +187,25 @@ public function getNearestOutwardCode(string $outwardcode, int $limit = 10, int
         return collect($this->getResponse("outcodes/$outwardcode/nearest?limit=$limit&radius=$radius"));
     }
 
+    /**
+     * Get nearest postcodes for a given longitude & latitude
+     *
+     * @param float $longitude
+     * @param float $latitude
+     *
+     * @return object
+     *
+     * @throws \GuzzleHttp\Exception\GuzzleException
+     */
+    public function nearestPostcodesForGivenLngAndLat(float $longitude, float $latitude): object
+    {
+        return collect($this->getResponse(sprintf(
```

Comment:
If we are going to wrap this in `collect()` then the return type will need adjusting to return the correct type